### DISCUSSION TITLE
Is 3D graphics acceleration on the roadmap? Approximately how far away is it?

### DISCUSSION THREAD

**Discussion by vesper8** [utmapp/UTM#3366] [General]
I'm asking in regards to running Windows 10/11 on Mac, specially a M1 Mac.

I'm wondering if this is a matter of time or if there are roadblocks that seem near impossible to surmount? 3D graphics acceleration is something that Parallels does very well and I would like to know if we can look forward to something similar being possible on UTM or if it's going to take years at best?

Many thanks

**Comment by osy** [comment]
It’s supported by VirGL which currently only has linux driver support. In theory if someone creates Windows drivers, it should work. But afaik there’s no active projects for that I know of.
